components/crm/internal/adapters/mongodb/alias/alias_query.mongodb.go

```
@@ -0,0 +1,230 @@
+	if err != nil {
+		libOpenTelemetry.HandleSpanError(spanFind, "Invalid metadata value", err)
+		return nil, err
+	}
+
+	cursor, err := coll.Find(ctx, filter, opts)
+	if err != nil {
+		libOpenTelemetry.HandleSpanError(spanFind, "Failed to find aliases", err)
+
+		return nil, err
+	}
+	defer func() {
```

golangci-lint (2.11.3)

[error] missing whitespace above this line (no shared variables above defer)

(wsl_v5)